pyproject.toml

```diff
@@ -28,7 +28,7 @@ deep-translator = 'deep_translator.__main__:main'
 dt = 'deep_translator.__main__:main'
 
 [tool.poetry.dependencies]
-python = "^3.7"
+python = ">=3.8,<4.0"
```

Comment:
This would make deep-translator dependent on python3.8, which is not a good idea

Reply:
ok understood. But most of the packages todays are dropping support for python 3.7. But anyway swtching back to python 3.7

Reply:
I think I understand here what you wanted to do. So the pre-commit seems to work with python3.8 and that is fine, but why did you update the dependency of the whole package to python3.8

You can make the pre-commit work with python3.8 without updating the dependencies of the package, because pre-commit is only a dev dependency. This is why I was skeptical about this